src/dotnet/BuiltInCommandsCatalog.cs

```diff
@@ -59,7 +59,7 @@ public static class BuiltInCommandsCatalog
             },
             ["migrate"] = new BuiltInCommandMetadata
             {
-                Command = MigrateCommand.Run,
+                Command = MigrateCommandCompose.Run,
```

Comment:
It is the composition root of Migrate Command http://blog.ploeh.dk/2011/07/28/CompositionRoot/

Or factory? But factory is too board